spec/lib/tasks/tmp_swap_travel_advice_link_spec.rb

```diff
@@ -71,4 +71,46 @@
              bulk_publishing: true,
            ))
   end
+
+  context "another item doesn't have any links" do
+    let(:no_links_edition) do
+      {
+        content_id: SecureRandom.uuid,
+        document_type: "travel_advice",
+        publishing_app: "travel-advice-publisher",
+      }
+    end
```

Comment:
Don't you also have to stub the publishing-api call too `get_editions` for `no_links_edition` as well?